acquia_lift.module

```diff
@@ -39,3 +38,38 @@ function acquia_lift_page_attachments(array &$page) {
   // Attach lift CSS to all pages.
   $page['#attached']['library'][] = 'acquia_lift/acquia-lift';
 }
+
+/**
+ * Implements hook_toolbar().
+ */
+function acquia_lift_toolbar() {
+  $items = [];
+
+  $items['acquia_lift'] = [
+    '#type' => 'toolbar_item',
+    'tab' => [
+      '#type' => 'html_tag',
+      '#tag' => 'button',
+      '#value' => t('Acquia Lift'),
+      '#attributes' => [
+        'title' => t('Acquia Lift'),
+        'class' => ['toolbar-icon', 'toolbar-icon-acquia-lift'],
+        // This particular id will make sure the JS behavior is attached to the
+        // link.
+        'id' => ['openLiftLink'],
+        'aria-pressed' => 'false',
+      ],
+    ],
+    '#access' => AccessResult::allowedIfHasPermission(\Drupal::currentUser(), 'access acquia lift links'),
```

Comment:
No problem. PathContext service's shouldAttach() currently checks two things: 1) does Lift has good credential? 2) is current page URL suitable for loading Lift? If yes to both, it loads lift.js (and everything else). I think we should use shouldAttach() here as well, because the shortcut only works if we know lift.js actually loads on the page.